### DISCUSSION TITLE
Masked mail with JMAP

### DISCUSSION THREAD

**Discussion by HelloFillip** [stalwartlabs/jmap-server#65] [General]
It's been a while since I looked into the JMAP specification work. I'm using masked mail heavily and want to swap out my current JMAP service with Stalwart, but I cannot find anything regarding masked mail, either in Stalwart itself, or in the IETF specification work anymore. 

I'd appreciate if anyone can help direct me to the current situation so I can understand where to start.

**Comment by mdecimus** [comment]
Are you referring to the masked email feature offered by Fastmail? This is not part of the JMAP specs, it is a feature they have implemented separately.
Since with Stalwart you have complete control of the server, you could create masked emails manually or dynamically using Sieve scripts.

**Reply by HelloFillip** [reply]
Thanks. This is where I was struggling: the last I read, from both 1Password and Fastmail, masked mail was in the JMAP standardization process, but I've never seen it in the actual specifications. 

This has been very helpful. 🙏
